src/mpid/ch4/netmod/ofi/ofi_finalize.c

```diff
@@ -7,15 +7,89 @@
 #include "ofi_impl.h"
 #include "ofi_noinline.h"
 
+#define MPIDI_OFI_FLUSH_CONTEXT_ID 0xF000
+#define MPIDI_OFI_FLUSH_TAG        1
```

Comment:
This could debugging misbehaving apps hard if this message is accidentally received by the user. Using a reserved value would be a good idea.

Reply:
Well, it is already covered by the `MPIDI_OFI_FLUSH_CONTEXT_ID`. No tag can guarantee collision free since this is out-of-band.